traffic_ops_ort/t3c/torequest/torequest.go

```diff
@@ -274,6 +277,14 @@ func (r *TrafficOpsReq) atsTcExecCommand(cmdstr string, queueState int, revalSta
 		return nil, errors.New("Error from atstccfg: " + err.Error() + ": " + errbuf.String())
 	}
 
+	if errStr := errbuf.String(); len(errStr) != 0 {
+		log.Warnln(`atstccfg stderr start
+` + errStr + `
+atstccfg stderr end`)
+	} else {
+		log.Warnln(`atstccfg stderr was empty`)
+	}
+
```

Comment:
needs `go fmt`

Reply:
This has `gofmt`. I think what you're seeing is the multi-line string literal? That's normal.

Reply:
oh yeah. Just looked wonky at a glance.